superset-frontend/src/components/Datasource/CollectionTable.tsx

```diff
@@ -415,7 +415,7 @@ export default class CRUDCollection extends PureComponent<
       )),
     );
     if (allowAddItem) {
-      tds.push(<td key="add" />);
+      tds.push(<td key="add" aria-label="dd" />);
```

Comment:
Was this supposed to be `aria-label="Add"`?

Reply:
Oops, typo!